libraries/AP_Mount/AP_Mount_Gremsy.cpp

```diff
@@ -304,6 +322,11 @@ void AP_Mount_Gremsy::send_gimbal_device_set_attitude(float roll_rad, float pitc
         return;
     }
 
+    if (_link == nullptr) {
+        return;
+    }
+    const mavlink_channel_t _chan = _link->get_chan();
```

Comment:
Let's replace "_chan" with "chan" 'cuz normally the underscore is used to signify that the variable is an internal variable.  I don't mind a little more code churn.

Reply:
Done